app.js

```diff
@@ -7,20 +7,19 @@ const provider = new ethers.providers.JsonRpcProvider(network_url);
 const signer = new ethers.Wallet(account_private_key, provider);
 const apiKey = process.env.POLYGONSCAN_API_KEY
 
-const oracleFacingContractAddress = "0xC324b5aeA9d4b8c5E13891633A36E75cc6c03b26"
-// const linkContractAddress = "0xb0897686c545045aFc77CF20eC7A532E3120E0F1"
-const dataProviderAddress = "0xbf405c325841545033920F59E9Aa9fe3d9c3c517"
-const divaDiamondAddress = "0xFf7d52432B19521276962B67FFB432eCcA609148"
+const oracleFacingContractAddress = process.env.ORACLE_FACING_CONTRACT_ADDRESS
```

Comment:
For better transparency, wouldn’t it better have the addresses and graph url directly in the file rather than hiding it in .env?